app/src/main/java/org/mozilla/fenix/tabstray/NewTabButtonBinding.kt

```diff
@@ -0,0 +1,99 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.tabstray
+
+import android.view.View
+import android.widget.ImageButton
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.cancel
+import kotlinx.coroutines.flow.collect
+import kotlinx.coroutines.flow.map
+import mozilla.components.lib.state.ext.flowScoped
+import mozilla.components.support.base.feature.LifecycleAwareFeature
+import mozilla.components.support.ktx.kotlinx.coroutines.flow.ifAnyChanged
+import org.mozilla.fenix.R
+import org.mozilla.fenix.tabstray.browser.BrowserTrayInteractor
+import org.mozilla.fenix.tabstray.syncedtabs.SyncedTabsInteractor
+import org.mozilla.fenix.utils.Settings
+
+class NewTabButtonBinding(
+    private val store: TabsTrayStore,
+    private val settings: Settings,
+    private val newTabButton: ImageButton,
+    private val browserTrayInteractor: BrowserTrayInteractor,
+    private val syncedTabsInteractor: SyncedTabsInteractor
+) : LifecycleAwareFeature {
+
+    private var scope: CoroutineScope? = null
+
+    @OptIn(ExperimentalCoroutinesApi::class)
+    override fun start() {
+        scope = store.flowScoped { flow ->
+            flow.map { it }
+                .ifAnyChanged { state ->
+                    arrayOf(
+                        state.selectedPage,
+                        state.syncing
+                    )
+                }
+                .collect { state ->
+                    setFab(state.selectedPage, state.syncing)
+                }
+        }
+    }
+
+    override fun stop() {
+        scope?.cancel()
+    }
+
+    private fun setFab(selectedPage: Page, syncing: Boolean) {
+        /* Do not show fab when accessibility service is enabled */
```

Comment:
This binding is coupled with the `FloatingActionButtonBinding`, so let's reference in this comment here, or better as a kdoc for the class, to take a look at the other one?

Related: I've been using the symbol matching in KDocs to ensure that related classes can be found through Android Studio and other symbolic referencing tools, like rename, safe-delete, etc.

You can reference the class in the KDocs with `[FloatingActionButtonBinding]` and you should be able to control-click on it to go to that class from the docs.

Reply:
Sure, I'll move this comment on both class to the top in KDocs.  They will reference each other 👍